web/js/util/load.js

```diff
@@ -36,8 +36,7 @@ export default (function () {
           loading -= 1;
           if (loading === 0) {
             clearTimeout(indicatorTimeout);
-            indicatorTimeout = null;
```

Comment:
Was this line causing problems? I wonder if it was in there for browser compatibility?

Reply:
After testing, the fix is related to the next line in the code. Passing in a null parameter causes issues with the qvui hide function.

Reply:
The next line uses `indicatorId` not `indicatorTimeout`